lib/waylon/jenkins/job/rest.rb

```diff
@@ -70,7 +74,7 @@ def investigating?
         end
 
         def description
-          @job_build_description ||= @client.job.get_build_details(@name, last_build_num)['description']
+          @job_build_description ||= job_build_details['description']
```

Comment:
Good point. We shouldn't trust the underlying ruby jenkins api module. What's a good ruby way of protecting for this?